references/recaptcha-enterprise/deliver-token-v1/apiproxy/proxies/default.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2022 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<ProxyEndpoint name="default">
+    <PreFlow name="PreFlow">
+        <Request>
+            <Step>
+                <Name>AM-DeliverRecaptchaToken</Name>
+            </Step>
+        </Request>
+    </PreFlow>
+    <PostFlow name="PostFlow"/>
+    <HTTPProxyConnection>
+        <BasePath>/web/v1/token</BasePath>
```

Comment:
Same as before. Please add a `recaptcha-` prefix to this basepath.

Reply:
Yes: base path is has been set to: /recaptcha/v1/token